daemon/src.cmake

```diff
@@ -301,7 +301,7 @@ set(QCOMMONLIST
     ${ENGINE_DIR}/sys/con_common.cpp
 )
 
-if (NOT APPLE)
+if (NOT APPLE AND (USE_CURSES OR WIN32))
```

Comment:
Still, if the user wants to override it, I don't see why not. It would also make these checks easier and more consistent.